src/content/upgrades/merge/index.md

```diff
@@ -116,6 +116,10 @@ The Merge comes with changes to consensus, which also includes changes related t
 For more information, check out this blog post by Tim Beiko on [How The Merge Impacts Ethereum’s Application Layer](https://blog.ethereum.org/2021/11/29/how-the-merge-impacts-app-layer/).
 </ExpandableCard>
 
+## When will The Merge happen? {#wen-merge}
```

Comment:
```suggestion
## What date is The Merge? {#wen-merge}
```

@samajammin I think adding 'merge date' to the heading makes a bit more sense here.